src/yaml/ulissewolf/HRW Modding/ulissewolf-hybrid-railway-hrw-subway-converter.yaml

```diff
@@ -0,0 +1,21 @@
+group: "ulissewolf"
+name: "hybrid-railway-hrw-subway-converter"
+version: "1.0"
+subfolder: "700-transit"
+assets:
+- assetId: "ulissewolf-hybrid-railway-hrw-subway-converter"
+dependencies:
+- "network-addon-mod"
```

Comment:
The NAM cannot be automatically installed as a dependency. Just mention in the description that the Network Addon Mod is needed.